src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -532,7 +537,22 @@ private function getPropertySchema(PropertyMetadata $propertyMetadata, \ArrayObj
 
         $valueSchema = $this->getType($builtinType, $isCollection, $className, $propertyMetadata->isReadableLink(), $definitions, $serializerContext);
 
-        return new \ArrayObject((array) $propertySchema + $valueSchema);
+        return new \ArrayObject((array) $propertySchema + $valueSchema + $attributeSchema);
+    }
+
+    /**
+     * @param array $attributes
+     *
+     * @return array
+     */
+    private function getPropertyAttributes(array $attributes): array
```

Comment:
Is this new method necessary?

Reply:
My thought here was to have that method there to add additional functionality in the future. In case you wanted to do more with the "attributes" on properties.